src/components/autocompleteInput.js

```diff
@@ -679,6 +679,9 @@
         labelPropertyPath.length > 0 && labelPropertyPath[0] !== '';
       if (emptyPropertyPath) {
         optionLabel = labelPropertyPath.reduce((acc, propertyId) => {
+          if (!acc) {
```

Comment:
You define `acc` as `option` (line 686). Is it possible that `option` can be `null`/?